pkgs/applications/version-management/git-and-tools/git-team/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = null;
+
+  proxyVendor = true;
+
```

Comment:
This is a no-op with `vendorSha256 = null`
```suggestion
```

Reply:
I don't think so. At least, building the package fails when omitting proxyVendor:
```
error: builder for '/nix/store/5zlpw3hqp966wq29qf487qhrcgc4157p-git-team-1.7.0.drv' failed with exit code 1;
       last 10 log lines:
       > configuring
       > building
       > Building subPackage .
       > go: inconsistent vendoring in /private/tmp/nix-build-git-team-1.7.0.drv-0/source:
       >        github.com/fatih/color@v1.11.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
       >        github.com/urfave/cli/v2@v2.3.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
       >
       >        To ignore the vendor directory, use -mod=readonly or -mod=mod.
       >        To sync the vendor directory, run:
       >                go mod vendor
       For full logs, run 'nix log /nix/store/5zlpw3hqp966wq29qf487qhrcgc4157p-git-team-1.7.0.drv'.
Traceback (most recent call last):
  File "/nix/store/q369ksyp8jr1ng9fjgb6vaj61wzkk4j4-nix-update-0.5.0/bin/.nix-update-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/q369ksyp8jr1ng9fjgb6vaj61wzkk4j4-nix-update-0.5.0/lib/python3.9/site-packages/nix_update/__init__.py", line 199, in main
    nix_build(options)
  File "/nix/store/q369ksyp8jr1ng9fjgb6vaj61wzkk4j4-nix-update-0.5.0/lib/python3.9/site-packages/nix_update/__init__.py", line 153, in nix_build
    run(
  File "/nix/store/q369ksyp8jr1ng9fjgb6vaj61wzkk4j4-nix-update-0.5.0/lib/python3.9/site-packages/nix_update/utils.py", line 35, in run
    return subprocess.run(
  File "/nix/store/z740mapbgfi953b61jx72c0jy3phfcy8-python3-3.9.10/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['nix', 'build', '--experimental-features', 'nix-command', '-f', './.', 'git-team']' returned non-zero exit status 1.
```

Reply:
This is actually a bug, I'll open a PR to correct it.

Reply:
Only works if sandboxing is disabled (default on darwin), fails if it is enabled (default on linux).
```
nix-build -A git-team --option sandbox true
this derivation will be built:
  /nix/store/l4hbz1p5v7qb2jszdq1k7dvadiab0spg-git-team-1.7.0.drv
building '/nix/store/l4hbz1p5v7qb2jszdq1k7dvadiab0spg-git-team-1.7.0.drv'...
unpacking sources
unpacking source archive /nix/store/df3ry1f7dfvvda1xiyw72hc0l479z0rk-source
source root is source
patching sources
configuring
building
Building subPackage .
go: github.com/fatih/color@v1.11.0: Get "https://proxy.golang.org/github.com/fatih/color/@v/v1.11.0.mod": dial tcp: lookup proxy.golang.org: no such host
go: downloading github.com/urfave/cli/v2 v2.3.0
go: downloading github.com/fatih/color v1.11.0
go: github.com/fatih/color@v1.11.0: Get "https://proxy.golang.org/github.com/fatih/color/@v/v1.11.0.mod": dial tcp: lookup proxy.golang.org: no such host
error: builder for '/nix/store/l4hbz1p5v7qb2jszdq1k7dvadiab0spg-git-team-1.7.0.drv' failed with exit code 1;
       last 10 log lines:
       > unpacking source archive /nix/store/df3ry1f7dfvvda1xiyw72hc0l479z0rk-source
       > source root is source
       > patching sources
       > configuring
       > building
       > Building subPackage .
       > go: github.com/fatih/color@v1.11.0: Get "https://proxy.golang.org/github.com/fatih/color/@v/v1.11.0.mod": dial tcp: lookup proxy.golang.org: no such host
       > go: downloading github.com/urfave/cli/v2 v2.3.0
       > go: downloading github.com/fatih/color v1.11.0
       > go: github.com/fatih/color@v1.11.0: Get "https://proxy.golang.org/github.com/fatih/color/@v/v1.11.0.mod": dial tcp: lookup proxy.golang.org: no such host
       For full logs, run 'nix log /nix/store/l4hbz1p5v7qb2jszdq1k7dvadiab0spg-git-team-1.7.0.drv'.
```